mico-core/src/main/java/io/github/ust/mico/core/broker/DeploymentBroker.java

```diff
@@ -235,27 +293,23 @@ public void checkIfKafkaEnabledServiceIsDeployable(MicoServiceDeploymentInfo mic
         }
     }
 
-    private MicoServiceDeploymentInfo buildMicoService(MicoServiceDeploymentInfo serviceDeploymentInfo) {
-        MicoService micoService = serviceDeploymentInfo.getService();
+    private MicoService buildMicoService(MicoService micoService) {
```

Comment:
Maybe add javadoc explaining how the service object is altered by this method and why it can't be saved in this method.